src/Tribe/Notices.php

```diff
@@ -16,6 +16,17 @@ class Tribe__Notices {
 	 * @return bool
 	 */
 	public static function set_notice( $key, $notice ) {
+
+		/**
+		 * Make notices filterable.
```

Comment:
(Though, we know the version number so you could add it directly rather than using the M17.11 placeholder)